apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/OpenAPIContentProvider.java

```diff
@@ -0,0 +1,292 @@
+  public static final String EOB_CLAIM_TYPE_VALUE =
+      """
+    A list of claim types that filters the response to contain only EoBs which have any of the specified claim types.
+    This is optional and defaults to _*_ (all types);
+    Supported Claim Type values:
+       - `carrier`
+       - `inpatient`
+       - `outpatient`
+       - `hha`
+       - `hospice`
+       - `snf`
+       - `dme`
+       - `partd`
+       - `inpatient`
+       - `inpatient`
+       - `inpatient`
+       - `inpatient`
+
+    Examples:
+       - `type=carrier,inpatient,snf,dme`
+       - `type=outpatient`
+       - `type=*;`""";
+
+  /** Open API short description for /ExplanationOfBenefit's serviceDate parameter. */
+  public static final String EOB_SERVICE_DATE_SHORT =
+      "Filter claims by the _billable period_. In other words, filter claims whose _through date_ fall within the given range";
+  /** Open API content value for /ExplanationOfBenefit's serviceDate parameter. */
+  public static final String EOB_SERVICE_DATE_VALUE =
+      """
+   Only satisfy the Search request if a claim's _billable period_.
+   falls within a specified _DateRange_. A DateRange can include both
+   low and high date values, only a low date value, or only a high date value.
+   Examples:
+     - `&service-date=gt2023-01-02&service-date=lt2023-05-01`
+     - `&service-date=gt2023-01-02`
+     - `&_service-date=lt2023-05-01`""";
+
+  /** Open API short description for /ExplanationOfBenefit's excludeSAMSHA parameter. */
+  public static final String EOB_EXCLUDE_SAMSHA_SHORT =
+      "If _true_, exclude any (all) SAMHSA-related claims";
+  /** Open API content value for /ExplanationOfBenefit's excludeSAMSHA parameter. */
+  public static final String EOB_EXCLUDE_SAMSHA_VALUE =
+      """
+   The _Substance Abuse and Mental Health Services Administration_ (SAMHSA)
+   is the agency within the U.S. Department of HHS that leads public health efforts to advance the behavioral health of the nation.
+   Setting this flag to _true_, modifies the request to filter out all SAMSHA-related claims from the response.
+   Examples:
+      - `&excludeSAMHSA=true`""";
+
+  /** Open API short description for /ExplanationOfBenefit's includeTaxNumbers parameter. */
+  public static final String EOB_INCLUDE_TAX_NUMBERS_SHORT =
+      "If `true`, tax numbers are included in the results.";
+  /** Open API content value for /ExplanationOfBenefit's includeTaxNumbers parameter. */
+  public static final String EOB_INCLUDE_TAX_NUMBERS_VALUE =
+      """
+   Setting this flag to _true_, provides tax number in the EOB transformed data for the response.
+   Example:
+      - `&includeTaxNumbers=true`""";
+
+  /** Open API content short description for partially adjudicated claim MBI ID to be used. */
+  public static final String PAC_MBI = "The patient identifier to search for.";
+  /** Open API content value for partially adjudicated claim MBI ID to be used. */
+  public static final String PAC_MBI_VALUE =
```

Comment:
I'm not sure why this isn't already the case, but the HAPI FHIR OpenAPI resources aren't flagging this as a required parameter. We need to indicate that it is in the content here in some _standard_ way at the beginning of the description, e.g.
```
**NOTE: THIS IS A REQUIRED FIELD***
```

I assume that this is not an isolated incident.